src/mono/wasm/debugger/tests/debugger-test-with-source-link/source-link.json

```diff
@@ -0,0 +1,5 @@
+{
+    "documents": {
+        "/fakepath/*":               ""

```

Comment:
Does `GetDataAsync` still get called, given the empty string here?

Reply:
Yes, that is why I detected that the code there was wrongly using `SourceUri.LocalPath`

Reply:
What is SourceUri, and SourceLinkUri in this case?